core/templates/dev/head/components/state-directives/outcome-editor/outcome-editor.directive.ts

```diff
@@ -59,7 +59,9 @@ angular.module('oppia').directive('outcomeEditor', [
             ENABLE_PREREQUISITE_SKILLS, INTERACTION_SPECS) {
           var ctrl = this;
           ctrl.editOutcomeForm = {};
-          ctrl.isInQuestionMode = StateEditorService.isInQuestionMode;
+          ctrl.isInQuestionMode = function() {
```

Comment:
This is odd. Why does it make a difference?

Reply:
It was giving undefined earlier. Besides, I feel like the earlier version is unnecessary. In all other parts of the code we do the new form for isInQuestionMode but only in this file do we do what it was earlier.

Reply:
Thanks!

To follow up, I checked with @bansalnitish and the issue is that "this" is defined differently in the two invocations. In one case, the "this" refers to "ctrl", and in the other case it refers to "StateEditorService".

/cc @aks681